interactive_engine/executor/ir/graph_proxy/src/utils/expr/eval.rs

```diff
@@ -54,6 +57,46 @@ pub enum Operand {
     VarMap(Vec<Operand>),
 }
 
+impl Operand {
+    /// only get the PropId, else None
+    pub(crate) fn get_var_prop_id(&self) -> GraphProxyResult<PropId> {
```

Comment:
Do we need both `get_var_prop_id` and `try_get_var_prop_id `? It's better to preserve one of them.

Reply:
remove `try_get_var_prop_id `.